tests/test_fields.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+import unittest
+from coaster.sqlalchemy import BaseMixin
+from coaster.utils import LabeledEnum
+from coaster.db import db
+from sqlalchemy import Column, Unicode
+from baseframe import __
+from baseframe import baseframe
+import baseframe.forms as forms
+import baseframe.forms.sqlalchemy as forms_sqlachemy
+from .fixtures import app1 as app
+
+
+class MY_LENUM(LabeledEnum):
+    FIRST = (1, 'first', __("First"))
+    SECOND = (2, 'second', __("Second"))
+    THIRD = (3, 'third', __("Third"))
+
+
+class MyForm(forms.Form):
+    position = forms.EnumField(__("Position"), choices=MY_LENUM, default=MY_LENUM.THIRD)
```

Comment:
Your test is missing a test for the `default`. This particular usage will fail, since there’s no translation from value to name.

Reply:
Can we do this? 
```
class EnumField(SelectField):
    def __init__(self, *args, **kwargs):
        super(EnumField, self).__init__(*args, **kwargs)
        self.lenum = self.choices
        self.choices = self.choices.nametitles()
        if self.default:
            self.default = self.lenum.get(self.default).name
```

Reply:
Let's do this:

1. Rename the field to `LabeledEnumField`
2. Change the parameter to `lenum`. If `lenum` is provided, `self.choices` is set from it.
3. `default = kwargs.pop('default')` (before calling `super`, followed by) `self.default = self.lenum[default].name`

Reply:
Using `lenum` instead of `choices` is analogous to how `QuerySelectField` accepts `query` instead of `choices`.

Reply:
Actually, strike `LabeledEnumField`. For consistency this should be `EnumSelectField`.